web-components/src/types/rdf/index.ts

```diff
@@ -12,3 +12,11 @@ export interface NameUrl {
   'schema:name': string;
   'schema:url': URL;
 }
+
+export interface URLList {
+  '@list': URL[];
+}
+
+export interface StringList {
+  '@list': URL[];
```

Comment:
mmm... StringList.. `'@list': string[];` ?

Reply:
in any case, I think StringList is not used anywhere

Reply:
removed